services/121-service/src/user/user.controller.ts

```diff
@@ -48,26 +48,20 @@ export class UserController {
     const _user = await this.userService.findOne(loginUserDto);
     if (!_user) {
       throw new HttpException('Unauthorized', HttpStatus.UNAUTHORIZED);
-    } else if (_user.status == 'inactive') {
-      throw new HttpException(
-        'Account deactivated. Contact organization administration.',
-        HttpStatus.UNAUTHORIZED,
-      );
     }
 
     const token = await this.userService.generateJWT(_user);
-    const { email, role, status } = _user;
+    const { email, roles } = _user;
```

Comment:
I think the only thing that needs to be returned to the interface is the `token`, as that contains all the data we'd need already (in a verifiable way).
And the other data isn't used.

So the only thing to return would be:
```
{ 
  token: '...'
}
```